machine-learning/tutorials/SentimentAnalysis/Program.cs

```diff
@@ -79,17 +79,17 @@ public static ITransformer BuildAndTrainModel(MLContext mlContext, IDataView spl
             // This is used to format and clean the data.  
             // Convert the text column to numeric vectors (Features column) 
             // <SnippetFeaturizeText>
-            var pipeline = mlContext.Transforms.Text.FeaturizeText(outputColumnName: DefaultColumnNames.Features, inputColumnName: nameof(SentimentData.SentimentText))
+            var estimator = mlContext.Transforms.Text.FeaturizeText(outputColumnName: "Features", inputColumnName: nameof(SentimentData.SentimentText))
             //</SnippetFeaturizeText>
-            // Adds a FastTreeBinaryClassificationTrainer, the decision tree learner for this project  
+            // append the training algorithm to the estimator
             // <SnippetAddTrainer> 
-            .Append(mlContext.BinaryClassification.Trainers.FastTree(numLeaves: 50, numTrees: 50, minDatapointsInLeaves: 20));
+            .Append(mlContext.BinaryClassification.Trainers.SdcaLogisticRegression(labelColumnName: "Label", featureColumnName: "Features"));
```

Comment:
>SdcaLogisticRegression [](start = 60, length = 22)

why not keep it to FastTree?

Reply:
Because SdcaLogisticRegression gets better results.